resource-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/resource/ScenarioPersistenceTest.java

```diff
@@ -87,7 +87,8 @@
     @Autowired
     private ResourceClientWrapper client;
 
-    private static final ConnectionDetail DETAIL = new SimpleConnectionDetail().uri("http://localhost:8082");
+    private static final ConnectionDetail DETAIL = new SimpleConnectionDetail().serviceName("test-data-service");

```

Comment:
Has to be changed back as it breaks the integration test
```suggestion
    private static final ConnectionDetail DETAIL = new SimpleConnectionDetail().serviceName("http://localhost:8082");

```